### DISCUSSION TITLE
Help me to make resource sub-navigation inside page

### DISCUSSION THREAD

**Discussion by acakluqman** [filamentphp/filament#11418] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

hello everyone, I'm just learning to use filament. I want to create a navigation menu on a page like the one below.

![localhost_8000-2024 02 15-14_09_07](https://github.com/filamentphp/filament/assets/25748055/e50874c1-577a-4c4e-b53a-82be9c6d9827)

Anyone can help me? or if there is a showcase example please give me the link...
thank you

**Comment by leandrocfe** [comment, marked as answer]
Take a look

https://filamentphp.com/docs/3.x/panels/clusters/

**Reply by kwekubright** [reply]
Hi @leandrocfe  
Thanks for sharing this link. 🙏 
I want some extra information. 
❓ How can i show the sub navigation on all resource pages? 
With Clusters i see the default behavior is to show the sub navigation on List page but on the view and edit pages the sub navigation is not shown.

How can i enable sub navigation on all pages.?

Thanks.

**Reply by leandrocfe** [reply]
Use `getSubNavigation()` in the pages. Check this video please
https://youtu.be/V909WVmyGMU?si=sHaeFvz07K0InAys&t=419

**Reply by kwekubright** [reply]
Awesome 👍 . This was very helpful.
Thank you.
